### PR TITLE
Bootstrap/responsive

### PR DESCRIPTION
**Pull request by kaveh163**

### CODE REVIEW

File: routes/user.js [hunk 1]
```diff
@@ -108,10 +119,15 @@ module.exports = function (app, passport) {
     app.get('/register', function (req, res) {
         const info = req.session.flash2;
         req.session.flash2 = [];
-        res.json({
-            // info: req.flash('info')
-            info: info
-        });
+        if(info === undefined) {
+            return res.send('undefined');
```
Comment (kaveh163) [inline routes/user.js:123]: ghghgggfgfgfhghf

File: public/index.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@media screen and (max-width: 576px) {
+    .fontRes {
+        font-size: 16px;
+    }
+    .nav-fix {
+        padding:10px;
+    }
+    .flex-container {
```
Comment (kaveh163) [inline public/index.css:8]: hftffhhgfg

**Comment by kambiz-frounchi** [comment]
Great job making it responsive!

File: public/index.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+@media screen and (max-width: 576px) {
```
Comment (kambiz-frounchi) [inline public/index.css:1]: looks good!

**Pull request merged**